server/service/base_client.go

```diff
@@ -63,7 +63,7 @@ func (bc *baseClient) parseResponse(verb, path string, response *http.Response,
 
 	bc.setServerCapabilities(response)
 
-	if responseDest != nil {
+	if responseDest != nil && response.StatusCode != http.StatusNoContent {
```

Comment:
What request returns 204?

Reply:
in this case it was `POST /api/fleet/orbit/disk_encryption_key`